src/Artsy/Router/buildServerApp.tsx

```diff
@@ -27,6 +28,7 @@ export function buildServerApp(config: RouterConfig): Promise<Resolve> {
       const historyMiddlewares = [queryMiddleware]
       const resolver = new Resolver(relayEnvironment)
       const render = createRender({})
+      const headTags = []
```

Comment:
`headTags` will hold any components that should be rendered in the head after SSR is complete. It's passed via boot into a context provider.